.infrastructure/cronjob.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: todoapp-cronjob
+  namespace: mateapp
+spec:
+  schedule: "*/4 * * * *"
+  concurrencyPolicy: Allow
```

Comment:
The `concurrencyPolicy` value should be capitalized as `Allow` instead of `allow`. This is important for the CronJob to interpret the policy correctly.